common/history/buildings/07_south_america.txt

```diff
@@ -1181,19 +1181,30 @@
 				level=1
 				reserves=1
 				activate_production_methods={ "pm_basic_shipbuilding" "pm_merchant_guilds_building_shipyards" "pm_military_shipbuilding_wooden" }
+			}
+				building="building_construction_sector"
```

Comment:
Something is wrong here, causes a bunch of errors with buildings in south america.

Error: untyped effect [ Expected opening bracket ]
  Script location: file: common/history/buildings/07_south_america.txt line: 1498

Reply:
Yup, looks like I deleted the create_building={ when moving things around. Fixed it.